demos/openshift-install/installer.sh

```diff
@@ -174,15 +174,37 @@ while [ "$1" != "" ]; do
 done
 
 if [ "$OPENSHIFT_URL" == "" ]; then
-  echo "Missing value in --ocp-url - exiting"
-  exit 1
+  read -p 'Please specify OpenShift URL to be configured:' OPENSHIFT_URL
+  if [ "$OPENSHIFT_URL" == "" ]; then
+    echo "Missing value in OpenShift URL - exiting"
+    exit 1
+  fi
 fi
 
 if [ "$PROJECT_NAME" == "" ]; then
-  echo "Missing value in --project-name - exiting"
-  exit 1
+  read -p 'Please specify project name to be configured:' PROJECT_NAME
+  if [ "$PROJECT_NAME" == "" ]; then
+    echo "Missing value in project name - exiting"
+    exit 1
+  fi
+fi
+
+if [ "$ACCOUNT_NAME" == "" ]; then
+  read -p 'Please specify account name to be configured (default):' ACCOUNT_NAME
```

Comment:
The standard is using [ ] for the default values

```suggestion
  read -p 'Please specify account name to be configured [default]: ' ACCOUNT_NAME
```

Reply:
Actually the standard for default is {} according to:
square brackets [optional option]
angle brackets <required argument>
curly braces {default values}
parenthesis (miscellaneous info) -

I will change it